### PR TITLE
add landing mybike

### PR DESCRIPTION
**Pull request by Vladyslav-amp**
[DEMO LINK]( https://Vladyslav-amp.github.io/layout_landing-page/ )

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good start 👍 
Let's improve your page
1. Need to fix header on desktop version
image

2. Add favicon on page
image

3. Need to remove scroll when this menu is open
image

**Review by vlad-demchuk — CHANGES_REQUESTED** [review]
1. there should be clickable hover effect for call icon

image
2. 

image

3. scroll isn't disabled

image
4. the button width doesn't match the design

image
5. contact us doesn't match the design

image
design 

image

6. you have spaces in textarea

image

7. all form fields should be required and cleared on form submission
8. phone link leads to 404, address should be a link as well

image

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Consider fixing validation errors: 
Screenshot 2023-08-06 at 16 40 14
Screenshot 2023-08-06 at 16 39 23
Consider forbidding the page to scroll and make the menu scrollable for the landscape mode.  
Screenshot 2023-08-06 at 16 38 41
The background-position for mobiles doesn't match the Figma design
Screenshot 2023-08-06 at 16 38 33
Check indentations here
Screenshot 2023-08-06 at 16 38 19
The background position for the desktop doesn't match the Figma design. 
Screenshot 2023-08-06 at 16 38 01
Consider making the link follow google maps 
Screenshot 2023-08-06 at 16 37 41
Consider removing outlines 
Screenshot 2023-08-06 at 16 37 33
This link doesn't work 
Screenshot 2023-08-06 at 16 37 22
Screenshot 2023-08-06 at 16 37 15
The form should be cleared after submission. 
Screenshot 2023-08-06 at 16 36 40
Consider hacking text styling for the textarea
Screenshot 2023-08-06 at 16 36 33
Consider adding a hover for this button

**Comment by Vladyslav-amp** [comment]
In my last commit I can't remove scroll in burger menu. Only imgs from categories scroll. I tried to fix it in many ways and it still not working. Can you give me a hint to fix it?

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Information on how to remove scrolling is available in the task checklist.
https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Add hover effects for these clickable elements
image

2. The user will not be able to stretch this window
image

3.  Need to fix the background image in footer
Demo
image

Design
image

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,283 @@
...
-    <h1>Landing Page</h1>
+  <!--#endregion-->
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <div class="top-bar__block">
+              <a href="#" class="top-bar__logo-link">
+                <img src="images/MyBike-white.svg"
```
Comment (anastasiiavorobiova) [inline src/index.html:26]:
```suggestion
                <img 
                  src="images/MyBike-white.svg"
```
Consider fixing indentations accordingly to the code style guide

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,283 @@
...
+                  alt="MyBike logo"
+                  class="top-bar__logo">
+              </a>
+            </div>
+            <div class="top-bar__icons">
+              <div class="top-bar__icons icon--slider">
+                <a href="tel:+123455555555"
```
Comment (anastasiiavorobiova) [inline src/index.html:33]: See my previous comment

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost perfect: Screenshot 2023-08-13 at 18 51 14
1) Consider fixing validation errors 
Screenshot 2023-08-13 at 18 47 54
2) Add menu scroll for landscape mode
Screenshot 2023-08-13 at 18 46 34
3) Check font styles here
Screenshot 2023-08-13 at 18 46 26
4) Consider fixing the background for autocomplete

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. When I move the cursor over these fields, they jump a little, this needs to be fixed
image

2. Add transition for hover effects
image
image

3. Need to fix position image on the mobile version
Demo
image

Design
image

4. The same as above
Demo
image

Design 
image

**Comment by Vladyslav-amp** [comment]
1. I tried to fix problem when moved cursor on input and it jump, nothing not help(
2. My header background for tablet mode doesn't fit, problem with width, its problem with original size of image or its my mistake? (added   background-size: cover ) img is too wide for tablet mode. 
3.  Footer problem the same like in header, for landscape mode img fit with all width, but when I tried to change size for tablet or phone, the image is too wide. I tried to write background-size: contain - that help, but I had vertical white space. 
Could you give me a hint or point out my mistake to me.

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
1. Need to remove border for hover effects for these fields
2. You can use background-position with percent

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Need to fix the burger menu, check it on the design
image

**Review by anastasiiavorobiova — APPROVED** [review]
Fantastic work! Looks awesome!

**Pull request closed without merging**